README.md

```diff
@@ -95,13 +95,13 @@ $ _build/default/examples/trees.exe
 The same thing can also be accomplished using `irmin`, the command-line application installed with `irmin-unix`, by running:
 
 ```bash
-$ echo "root=." > .irminconfig
+$ echo "root: ." > irmin.yaml
```

Comment:
typo: `irmin.yml`

Reply:
This has already been fixed in the latest update. See commit
7144d90